### PR TITLE
[FR-299] Change image format

### PR DESCRIPTION
**Pull request by hmdros**
Addressing performance improvement suggested by lighthouse reports changed this image to .webp format in internal PR.

### CODE REVIEW

**Comment by kindoflew** [comment]
Support for `webp` is pretty good (https://caniuse.com/webp), but I wonder if we should also have `jpeg` as a fallback, just in case? what do you think, @etagwerker?

**Comment by etagwerker** [comment]
@kindoflew I like your idea 👍

**Review by bronzdoc — APPROVED** [review]

**Comment by hmdros** [comment]
Thanks for the suggestions @kindoflew

I added this @supports CSS rule based on [this blog post](https://www.irigoyen.dev/blog/2021/02/11/image-fallbacks-in-css-using-supports/). Let me know if you have any other opinion.

Also moved back the .jpg images in FR pull request

File: vendor/assets/stylesheets/fastruby/styleguide/4-molecules/_location.scss [hunk 1]
```diff
@@ -2,16 +2,25 @@
   background-color: $grey;
   color: $white;
   padding: 13rem 1rem 6rem 1rem;
-  background-image: asset_url('bg-location-2.webp');
+  background-image: asset_url("bg-location-2.webp");
+  @supports (background-image: asset_url("bg-location-2.webp")) {
```
Comment (kindoflew) [inline vendor/assets/stylesheets/fastruby/styleguide/4-molecules/_location.scss:6]: i think this logic might be backwards. if the `@supports` is `true`, shouldn't the `.webp` image be nested here?

File: vendor/assets/stylesheets/fastruby/styleguide/4-molecules/_location.scss [hunk 2]
```diff
@@ -2,16 +2,25 @@
...
+    background-image: asset_url("bg-location-2.jpg");
+  }
   background-position: 50% 53%;
   margin: 3rem -1rem 0 -1rem;
   @include respond-above(xs) {
     background-position: 50% 53%;
-    background-image: asset_url('bg-location-2.webp');
+    background-image: asset_url("bg-location-2.webp");
+    @supports (background-image: asset_url("bg-location-2.webp")) {
```
Comment (kindoflew) [inline vendor/assets/stylesheets/fastruby/styleguide/4-molecules/_location.scss:14]: same as above.

File: vendor/assets/stylesheets/fastruby/styleguide/4-molecules/_location.scss [hunk 3]
```diff
@@ -2,16 +2,25 @@
...
+      background-image: asset_url("bg-location-2.jpg");
+    }
     margin: 0;
   }
   @include respond-above(sm) {
-    background-image: asset_url('bg-location.webp');
+    background-image: asset_url("bg-location.webp");
+    @supports (background-image: asset_url("bg-location.webp")) {
```
Comment (kindoflew) [inline vendor/assets/stylesheets/fastruby/styleguide/4-molecules/_location.scss:21]: same as above.

**Review by kindoflew — CHANGES_REQUESTED** [review]
Good idea with `@supports`! I think the logic needs to be flipped, but that's it!

**Comment by hmdros** [comment]
> Good idea with `@supports`! I think the logic needs to be flipped, but that's it!

Done! Thanks @kindoflew

**Pull request merged**